src/D2L.Security.OAuth2.TestFramework/AuthServiceMock.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Security.Cryptography;
+using System.Text.Json;

```

Comment:
The changes in tests here to add the using are because the JsonSerializer helper that we defined in the main package is now gone.
That means these tests are now using both System.Text.Json and Newtonsoft which is a bit ugly but I'm not worried about it.

If everything goes well I'll delete the references to Newtonsoft in the tests too.